sormas-app/app/src/main/java/de/symeda/sormas/app/contact/edit/ContactEditFragment.java

```diff
@@ -200,6 +207,14 @@ public void onLayoutBinding(FragmentContactEditLayoutBinding contentBinding) {
 				.setItems(DataUtils.toItems(Arrays.asList(ContactProximity.getValues((Disease) e.getValue(), ConfigProvider.getServerLocale()))));
 		});
 
+		contentBinding.contactMultiDayContact.addValueChangedListener(e -> {
```

Comment:
showing/hiding firstContact date can be solved in a more simple way using `app:dependencyParentField` and `app:dependencyParentValue` properties in the layout xml